### PR TITLE
feat: bump WsDFU to latest to include support for ExtendedIndexInfo

### PR DESCRIPTION
**Pull request by jeclrsg**
Signed-off-by: Jeremy Clements <Jeremy.Clements@lexisnexisrisk.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/comms/package.json [hunk 1]
```diff
@@ -69,10 +69,10 @@
         "tmp": "0.2.1"
     },
     "devDependencies": {
-        "@hpcc-js/bundle": "^2.11.3",
+        "@hpcc-js/bundle": "^2.10.23",
```
Comment (jeclrsg) [inline packages/comms/package.json:72]: When I was building this to test, this version number showed as incorrect; there doesn't seem to be a 2.11.3 for bundle

File: packages/comms/package.json [hunk 2]
```diff
@@ -69,10 +69,10 @@
...
         "@types/d3-request": "1.0.6",
         "@types/d3-time-format": "2.3.1",
-        "@types/node": "^10",
+        "@types/node": "^14",
```
Comment (jeclrsg) [inline packages/comms/package.json:75]: I don't believe this caused any problems, as I was able to build without warnings or errors. It was a required for a change in the "fs-extra" dependency, which was causing any of the "wsdl-..." npm scripts to fail

Comment (GordonSmith) [reply packages/comms/package.json:75]: "@types/node": "^16" would match what we are actually using?

Comment (GordonSmith) [reply packages/comms/package.json:75]: Looks like the CI failed because your package-lock file is out of sync (do a npm install and commit).

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Review by GordonSmith — APPROVED** [review]
(Waiting on tests to rerun)

**Pull request closed without merging**